### DISCUSSION TITLE
WebXR and Postprocessing

### DISCUSSION THREAD

**Discussion by IRobot1** [angular-threejs/angular-three#7] [Ideas]
The [following solution](https://twitter.com/Cody_J_Bennett/status/1482585611781480448)  was posted to get postprocessing to work in Web XR.

Here's a copy of the code that was posted.  Is it possible to add this support into the angular-three postprocessing renderer?

![image](https://user-images.githubusercontent.com/25032599/219878765-24bfef07-0f7b-4f81-bebd-428e0e3b8d94.png)

**Comment by nartc** [comment]
If it’s recommended in R3F then I can add it to NGT for sure. Thanks for bringing it up

**Comment by nartc** [comment]
I've released `angular-three-postprocessing@1.4.0` with this change. Can you please test it out? I can always revert it if it doesn't work

**Comment by IRobot1** [comment]
Still not working.  The console still includes warning

Note: The XRSession has completed multiple animation frames without drawing anything to the baseLayer's framebuffer, resulting in no visible output.

**Reply by nartc** [reply]
I didn't port the buffer size code over. Thought it wasn't needed. I'll push another release tmr. if it's still not working, then I need a reproduce with r3f version
